Bonsai.System/IO/GetEnvironmentVariable.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Reactive.Linq;
+
+namespace Bonsai.IO
+{ 
+    /// <summary>
+    /// Represents an operator that returns the value of an OS environment variable.
+    /// </summary>
+    /// 
+    [Description("Returns the value of an environment variable.")]
+    public class GetEnvironmentVariable : Source<string>
+    {
+        [Description("The name of the environment variable to query the value of.")]
+        public string Variable { get; set; }
+
+        /// <summary>
+        /// Generates an observable sequence containing a string with the value
+        /// of the queried environment variable.
+        /// </summary>
+        /// <returns>
+        /// An observable sequence containing the value of the queried
+        /// environment variable.
+        /// </returns>
```

Comment:
For consistency with documentation of similar operators, I would use something like:

```c#
        /// <summary>
        /// Gets the value of the specified environment variable for the current process
        /// and surfaces it through an observable sequence.
        /// </summary>
        /// <returns>
        /// A sequence containing the value of the specified environment variable. The
        /// value will be <see langword="null"/> if the environment variable is not found.
        /// </returns>
```